css/style.css

```diff
@@ -28,17 +28,23 @@ body::before
 }
 
 /* show signature download on hover */
-span.sig_btn > a:last-child
+span.sig_btn
 {
+    position: relative;
```

Comment:
That's needed so position absolute in the child works, right?
I never understood that crap.